tensorboard/webapp/widgets/range_input/range_input_component.ts

```diff
@@ -43,23 +35,6 @@ enum Position {
  *
  * Anatomy of the component:
  *
- * Single slider:
```

Comment:
:clap:  The logic was complicated enough to warrant a JSDoc diagram and you got to remove it all